### ISSUE TITLE
Fuzzy translations can be saved without edits

### ISSUE THREAD

**Issue by ocean90** [GlotPress/GlotPress-WP#1248]
Reported in https://meta.trac.wordpress.org/ticket/5788.

> I'm working on nl_NL-formal waiting strings and with this fuzzy, I didn't click "Accept" but clicked "Save"... And the translation changed to me instead of the original translator... I didn't change a single thing about the string...

**Comment by ocean90** [comment]
Maybe we can extend the check added in https://github.com/GlotPress/GlotPress-WP/commit/39e768afeba5c60d2f22c84ff413dd2cf129a31a to also include fuzzy strings.

https://github.com/GlotPress/GlotPress-WP/blob/0c395a7a8f37ab3b5ebafd2b239c74392e5177f9/gp-includes/routes/translation.php#L296-L310

**Comment by NekoJonez** [comment]
Thx for reporting it upstream... I personally find it tricky when it needs to reported here instead of the meta trac... 

(And it would be so awesome if a fix happens here, the meta trac ticket would be updated in a way but that is a whole other proces :P)

**Comment by psmits1567** [comment]
This is probably caused by the fact that you are PTE. Normally you cannot save a translation twice, and change the name of the original.
Or there has been a minor change in the original, and the old translated text is copied. So there is basically a new translation

**Comment by NekoJonez** [comment]
> This is probably caused by the fact that you are PTE. Normally you cannot save a translation twice, and change the name of the original.
> Or there has been a minor change in the original, and the old translated text is copied. So there is basically a new translation

Yeah, before I was PTE , I was able to fix fuzzy's by them ending up as new ones.
